docs/guides/usage.rst

```diff
@@ -30,42 +30,61 @@ General Usage
-    - :code:`--read-layout-adapters`: Path to a file with 3' adapter sequences (one sequence per line) used to identify adapter content.
-    - :code:`--read-layout-min-match-percentage`: Minimum percentage of reads containing an adapter for it to be considered as the library’s 3’-end adapter.
-    - :code:`--read-layout-min-frequency-ratio`: Minimum frequency ratio between the most and second most frequent adapters to select the 3’-end adapter.
-    - :code:`--library-source-min-match-percentage`: Minimum percentage of reads aligning with a library source for it to be considered representative of the library.
-    - :code:`--library-source-min-frequency-ratio`: Minimum frequency ratio between primary and secondary library sources, ensuring only the most prominent source is identified.
-    - :code:`--library-type-max-distance`: Maximum allowable distance between read pairs to classify the library type.
-    - :code:`--library-type-mates-cutoff`: Ratio cutoff to determine the consistency of mate orientation in paired-end reads.
-    - :code:`--read-orientation-min-mapped-reads`: Minimum number of mapped reads to ensure reliable inference of read orientation.
-    - :code:`--read-orientation-min-fraction`: Minimum fraction (must exceed 0.5) of reads supporting a given orientation to confirm its accuracy.
+   * - :code:`--output-directory`
+     - Path where output data will be saved. 
+   * - :code:`--temporary-directory`
+     - Path for storing temporary files generated during execution.
+   * - :code:`--cleanup-regime`
+     - Specifies which data should be kept after completion. Options are: :code:`DEFAULT, KEEP_ALL, KEEP_NONE, KEEP_RESULTS`
+   * - :code:`--verbosity`
+     - Controls the verbosity level of log output. Options are: :code:`DEBUG, INFO, WARN, ERROR, CRITICAL`
+   * - :code:`-h, --help`
+     - Show help screen and exit.
+   * - :code:`-v, --version`
+     - Show version information and exit.
 
-- **Processing and Performance Options**:
-    - :code:`--records`: Limits the number of input records to process; setting this to 0 will process all records.
-    - :code:`--threads`: Specifies the number of threads for concurrent processing to optimize performance.
-    - :code:`--tax-id`: Taxonomy ID for the sample source, aiding in organism-specific analyses.
 
-Meta Options
-------------
+.. list-table:: Processing and Performance Options
+   :widths: 40 60
 
-For help or version information, use the following:
+   * - :code:`--records`
+     - Limits the number of input records to process; setting this to 0 will process all records.
+   * - :code:`--threads`
+     - Specifies the number of threads for STAR to optimize performance.
+   * - :code:`--tax-id`
+     - Taxonomy ID for the sample source, aiding in organism-specific analyses.
 
-.. code-block:: bash
-
-   htsinfer --help
-   htsinfer --version
+.. list-table:: Library-specific Options
+   :widths: 40 60
```

Comment:
Make sure that long descriptions autowrap. Scrolling horizontally is annoying.

Reply:
Neither is this.. best I could do is add line breaks.